index.src.html

```diff
@@ -783,8 +783,14 @@ <h3 id="privacy-impact-assessment">
   the end-developer, system owners, or managers responsibility to realize
   this and act accordingly in the design of his/her system. Some use may
   warrant conducting as privacy impact assessment, especially when data
-  relating to individuals may be processed. Examples.
+  relating to individuals may be processed.
 
+  Specifications that expose such sensitive data should include a
+  recommendation that websites and applications adopting the API — but not
+  necessarily the implementing user agent — conduct a privacy impact assessment
```

Comment:
What do we mean by "not necessarily the implementing UA"? Why are we adding this?